app/components/ClueMode.tsx

```diff
@@ -902,6 +907,53 @@ export const ClueMode = ({ state, ...props }: ClueModeProps) => {
               }
             />
           </div>
+          <div>
+            <h3>Enumerations</h3>
+            <div>
+              <ButtonAsLink
+                text="Add Enumerations"
```

Comment:
Extremely minor nit but let's change the capitalization to match "Add an alternate solution" above (i.e. only first word capitalized). Either that or change the other to match this.